lib/sensu/server/process.rb

```diff
@@ -363,6 +362,10 @@ def process_check_result(result)
                 end
               end
             end
+            @logger.debug("updating result data", :check=> check)
+            result_truncated_output = result.dup
+            result_truncated_output[:output] = result[:output][0..256]
+            @redis.set("result:#{client[:name]}:#{check[:name]}", MultiJson.dump(result_without_output))
```

Comment:
How will a user clear this result table if they are sure they want to *really* remove a check?